torch/csrc/inductor/aoti_runtime/cpu_utils.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+// FIXME: Currently, CPU and CUDA backend are mutually exclusive.
+// This is a temporary workaround. We need a better way to support
+// multi devices.
+#ifndef USE_CUDA
+
+#define AOTI_RUNTIME_DEVICE_CHECK(EXPR)                    \
```

Comment:
Do we really need this for CPU? If not, I think make it define empty can save some uses of `USE_CUDA` later.

Reply:
Previously, I was thinking to have exactly the same empty macro to reduce the uses of ```USE_CUDA```, but changed my mind later. One rationale is that I want to reduce any surprise. For example, I feel turning something like ```AOTI_RUNTIME_DEVICE_CHECK(foo())``` to empty could be a surprise in cpu mode, because function foo may have both cuda and non-cuda side-effect. Instead, having a dedicated macro explicitly for each arch would make the code easier to interpret and reduce the chance of such surprises.